conda_build/environ.py

```diff
@@ -144,6 +144,7 @@ def get_dict(m=None, prefix=None):
         d['PKG_NAME'] = m.name()
         d['PKG_VERSION'] = m.version()
         d['PKG_BUILDNUM'] = str(m.build_number())
+        d['PKG_BUILD_STRING'] = str(m.build_id())
```

Comment:
>Maybe we could just pass the meta itself?

I'm not sure I understand the question.  You mean something like this?

```
d['PKG_BUILD_STRING'] = m.get_value('build/string', '')
```

I think `build_id()` is preferable because it provides a default value in some cases.